.gitignore

```diff
@@ -12,11 +12,10 @@
 # ocamlbuild working directory
 _build/
 
-# ocamlbuild targets
-*.byte
-*.native
-
-# oasis generated files
-setup.data
-setup.log
+# source control
```

Comment:
Do we still need these two?

Reply:
I normally put these as a matter of course in a repo - any reason not to put them?

Reply:
I don't have strong opinions. And it does no harm to leave them there

Reply:
The lines under `# OCaml` can be removed because all these files are under `_build/` which has an entry.